commit f36e4841660c183afd2d79b463afc55f62fef91e
Author: Vitaly Kuznetsov <vkuznets@redhat.com>
Date:   Wed Dec 7 01:16:26 2016 -0800

    hv: init percpu_list in hv_synic_alloc()
    
    commit 3c7630d35009e6635e5b58d62de554fd5b6db5df upstream.
    
    Initializing hv_context.percpu_list in hv_synic_alloc() helps to prevent a
    crash in percpu_channel_enq() when not all CPUs were online during
    initialization and it naturally belongs there.
    
    Signed-off-by: Vitaly Kuznetsov <vkuznets@redhat.com>
    Signed-off-by: K. Y. Srinivasan <kys@microsoft.com>
    Signed-off-by: Greg Kroah-Hartman <gregkh@linuxfoundation.org>

Index: linuxtest/dummy/rpi_1317_f36e4841660c183afd2d79b463afc55f62fef91e.txt
===================================================================
--- /dev/null
+++ linuxtest/dummy/rpi_1317_f36e4841660c183afd2d79b463afc55f62fef91e.txt
@@ -0,0 +1 @@
+dummy file to ensure patch has content.
